README.rst

```diff
@@ -1,7 +1,7 @@
 Welcome to Pyccel
 =================
-
-|build-status| |docs| |codacy|
+![.github/workflows/Github_pytes.yml](https://github.com/pyccel/pyccel/workflows/.github/workflows/Github_pytest.yml/badge.svg)
```

Comment:
typo: `Github_pytes.yml`->`Github_pytest.yml`

I think it would be better to use the same syntax as before which allows us to easily put all the badges on one line and specify the image attributes in one section at the end

Reply:
I'm just testing it to see if it works, apparently not

Reply:
I don't think it can work until the workflow was been run once though :/

Reply:
the problem is that it doesn't create the badge

Reply:
Yes that is what I would expect. The badge should point to the result of the workflow but if the workflow has never been run then there is no result to point to. Maybe you could try making it trigger on push to anything (so it runs once for this branch) then set it back to just push to master. That way a result exists and the badge has something to try and reference

Reply:
I have done what I suggested and the badge now exists as expected

Reply:
I tried what I suggested above. The badge now appears here:
https://github.com/pyccel/pyccel/workflows/master_tests/badge.svg